jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TargetImageParameters.java

```diff
@@ -123,4 +124,16 @@ public AuthParameters getAuth() {
   public void auth(Action<? super AuthParameters> action) {
     action.execute(auth);
   }
+
+  @Input
+  public boolean getEnablePlatformTags() {
+    if (System.getProperty(PropertyNames.ENABLE_PLATFORM_TAGS) != null) {
+      return Boolean.parseBoolean(System.getProperty(PropertyNames.ENABLE_PLATFORM_TAGS));
```

Comment:
What happens if a non-boolean value is provided? Should there be a specific validation?

Reply:
It will return false. I don't think there needs to be a specific validation as the invalid case would result in the default value which seems safe. I'm happy to update this if you know of a convenient way to check if a string is a valid boolean type. `BooleanUtils` from `commons-lang3` comes to mind but I'm not sure what the appetite is for adding a new dep?

Reply:
Yes, I think default `false` is good.